tests/test_enterprise/test_signals.py

```diff
@@ -171,7 +171,8 @@ def test_handle_user_post_save_with_pending_course_enrollment(
         assert PendingEnrollment.objects.count() == 0
         assert EnterpriseCourseEnrollment.objects.count() == 1
         mock_course_enrollment.return_value.enroll_user_in_course.assert_called_once_with(
-            user.username, course_id, 'audit', cohort=u'test_cohort'
+            user.username, course_id, 'audit', cohort=u'test_cohort',
```

Comment:
[style nitpick] I think this can be moved to multiple lines as Alex D has mentioned elsewhere in this PR.

Reply:
@alex-sheehan-edx It's done